packages/cashier/src/Components/confirm.jsx

```diff
@@ -32,27 +32,86 @@ const Row = ({ item_key, label, value }) => (
     </div>
 );
 
-const Confirm = ({ data, error, header, onClickBack, onClickConfirm }) => (
-    <div className='cashier__wrapper cashier__wrapper--confirm'>
-        <Icon data_testid='dti_confirm_details_icon' icon='IcConfirmDetails' width='128' height='128' />
-        <Text as='h2' color='prominent' align='center' weight='bold' className='cashier__header confirm__header'>
-            {header}
-        </Text>
-        <div className='confirm__column-wrapper'>
-            <div className='confirm__column'>
-                {data.map((d, key) => (
-                    <Row item_key={key} label={d.label} value={d.value} key={key} />
-                ))}
-            </div>
-        </div>
-        <div className='confirm__submit'>
-            <Button large text={localize('Back')} onClick={onClickBack} secondary />
-            <Button large text={localize('Confirm')} onClick={onClickConfirm} primary />
-        </div>
-        <FormError error={error} />
+const WarningBullet = ({ children }) => (
+    <div className='confirm__warnings-bullet-wrapper'>
+        <div className='confirm__warnings-bullet' />
+        {children}
     </div>
 );
 
+const Confirm = ({
+    data,
+    error,
+    header,
+    is_payment_agent_transfer_confirm_component,
```

Comment:
@george-usynin-binary Why not `is_payment_agent_transfer`? Since it's already a component and a confirm one.

Reply:
@Bahar-FS I wanted to emphasize that it is passed payment agent transfer component, not payment agent withdraw component. but i agree that `is_payment_agent_transfer` looks better and in general reflects the essence. will change it to `is_payment_agent_transfer`